rules/visual-design-specifications/rule.md

```diff
@@ -19,87 +19,90 @@ created: 2009-02-28T09:45:02.000Z
+**User story:** Storyboards are based on a specific scenario. The role that corresponds to that scenario must be clearly specified. A short text description of the scenario or story must clear enough that a team member or stakeholder could understand what is depicted before looking at the visuals.
 
-Start by identifying the key features and functionality that the product should have. This will help you focus your efforts and ensure that your mockup accurately represents your intended product.
+Also See: [Do you use User Stories format when appropriate?](https://www.ssw.com.au/rules/spec-do-you-use-user-stories/)
 
-::: img-large
-![Figure: Example of a hand-drawn mockup. Nice and quick for early concept design](hand-drawn-mockup.jpg)
-:::
+**Visuals:** The scenario is visually depicted in a sequential manner using sketches or photos. These images can range from quick, low-fidelity drawings to detailed, high-fidelity artifacts, depending on the purpose and intended audience of the storyboard. The visuals include relevant details such as the user’s environment or a sketch of the screen being interacted with.
 
-Then create a basic wireframe of your design. A wireframe is a visual representation of the layout and structure of your website or app. You can create digital wireframes using tools like Figma, Adobe XD, or Sketch. Alternatively, you can use paper and pencil to sketch out a rough wireframe by hand. Add details to your wireframe by adding boxes and labels for different features and elements (e.g. buttons, input fields, and images). 
+Also See: [Do you know the value of User Journey Mapping?](https://www.ssw.com.au/rules/user-journey-mapping/)
+
+**Captions:** Each visual should include corresponding s to describes the user’s actions, environment, device, and so on.
 
 ::: good
-![Figure: Good example – Example of wireframes (created in Figma)](figma_wireframe_app_screenshot.png)
+![Figure: Good example – a simple quick hand-drawn storyboards specific scenario with all 3  elements.](basic-storyboard.png)
 :::
 
-Once your low-fidelity design mockup has been approved, you can use it as a reference point for creating a high-fidelity mockup. 
+::: bad
+![Figure: Bad example – UML is virtually impossible to get clients to understand software specifications](bad-uml.jpg)
+:::
 
-### High-fidelity Mockups
+### How to create a storyboard
 
-A high-fidelity mockup is a more detailed version of a design that includes visual design elements such as typography, colors, and images. It is created using specialized design tools and takes more time to create than a low-fidelity mockup.
+Creating a storyboard can feel like a daunting task. However, remember they are most valuable as a low-fidelity artifact and don’t need to take a long time to create.
 
-::: greybox
```

Comment:
1. change from "greybox" to "info"

Reply:
<img width="850" alt="Screenshot 2023-09-04 at 15 01 49" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/4745861/6d4ca78d-6cf8-4d29-a868-bbf3ff5bb621">

@tiagov8 Hi Tiago, the info style seems off, am I using the right one?

Reply:
It is correct, but apparently a bug on "info" boxes if you have a list in the second line.

Can you change to:

> Storyboards help to create....
> 
> They can:
> 
> {{ LIST }}